modules/persistent-storage-additional-support-limitations-for-shared-resource-csi-driver.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: REFERENCE
+
+[id="persistent-storage-additional-support-limitations-for-shared-resource-csi-driver_{context}"]
+= Additional support limitations for the Shared Resource CSI driver
+
+[role="_abstract"]
+The Shared Resource CSI driver has the noteworthy limitations.
+
+* The driver is also subject to the limitations of CSI inline ephemeral volumes.
+
+* The value of the `readOnly` field must be `true`. Otherwise, on volume provisioning during pod startup, the driver returns an error to the kubelet. This limitation is in keeping with the upstream Kubernetes CSI Driver best practices for applying SELinux labels to associated volumes.
+
+* The driver ignores the `FSType` field because it only supports `tmpfs` volumes.
+
+* The driver ignores the `NodePublishSecretRef` field. Instead, it uses `SubjectAccessReviews` with the `use` verb to evaluate whether a pod can obtaining a volume that contains `SharedSecret` or `SharedConfigMap` resources.
+
+// [role="_additional-resources"]
+// .Additional resources
+//
+// * TBD link to upstream Kubernetes CSI Driver best practices for applying SELinux labels to associated volumes.
```

Comment:
Removed for now. We can add a link if the proposal is ratified. I also updated line 11 to say:
"* The value of the `readOnly` field must be `true`. Otherwise, on volume provisioning during pod startup, the driver returns an error to the kubelet. This limitation is in keeping with proposed best practices for the upstream Kubernetes CSI Driver to apply SELinux labels to associated volumes."

Reply:
@gabemontero WDYT?

Reply:
Updated. Thanks.